setup.cfg

```diff
@@ -24,7 +24,7 @@ install_requires =
     Click >= 6.7
     configshell-fb >= 1.1
     pycryptodomex >= 3.4.6
-    PyYAML < 5.1
+    PyYAML >= 5
```

Comment:
The spec file has

```
Requires:       python3-PyYAML >= 3.13
```

probably because Leap 15.1 has

```
$ rpm -q python3-PyYAML
python3-PyYAML-3.13-lp151.1.1.x86_64
```

Just thinking out loud - could the dependency information in `setup.cfg` be aligned with that in `ceph-bootstrap.spec` ?